Assets/MRTK/Core/Utilities/StandardShader/ClippingSphere.cs

```diff
@@ -25,7 +25,7 @@ public float Radius
             }
         }
 
-        private int clipSphereID;
+        protected int clipSphereID;
```

Comment:
Is there a reason this went from public to private

Reply:
Yes, I'm glad you asked! That is so users can extend these classes as outlined in the new documentation: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/6882e739656f792ee1c7c56cfd4a183ea46a843d/Documentation/Rendering/ClippingPrimitive.md

Reply:
can you add comment explaining what this should be? Since it's targeted to be extended. Same for others

Reply:
Good idea all! Added some comments.